packages/projection-typeorm/test/util.ts

```diff
@@ -3,7 +3,7 @@ import { createDatabase } from 'typeorm-extension';
 import { logger } from '@cardano-sdk/util-dev';
 
 export const connectionConfig = {
-  database: 'projection',
+  database: 'stake_pool',
```

Comment:
This is a generic util, having `stake_pool` as database name for all projection tests isn't great

Reply:
4f6d8619f , 4f6d8619f and c64200743 should solve this